tools/spocc/spocc_occ.xml

```diff
@@ -0,0 +1,93 @@
+<tool id="spocc_occ" name="Get species occurrences data" version="0.9.0">
+    <description>from GBIF, ALA, iNAT and others</description>
+    <requirements>
+        <requirement type="package" version="0.9.0">r-spocc</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript '$__tool_directory__/spocc_occ.R' 
+            '$sname' 
+            '$from' 
+            $limit; 
+
+        #if $from=='gbif':
+            sed -z -i -e 's/\n  \"/ /g' -e 's/\n}/ /g' output.tab
+        #end if
+    ]]></command>
+    <inputs>
+        <param name="sname" type="text" label="Scientific name" value="Genus species"/>
```

Comment:
maybe give an exampe?

Reply:
value should be empty right, but `help="e.g. Canis lupus" label="Scientific name of the genus species"`

Reply:
Yes I can improve this. 

Genus species is the format of a scientific name so `label="Scientific name of the genus species"` is kind of redundant. 
We should use smthing like `label="Scientific name of the species" help="Genus species format, eg : Canis lupus"`